### DISCUSSION TITLE
xscreensaver port to webgl for use with lively? (or similar)

### DISCUSSION THREAD

**Discussion by kenwud** [rocksdanister/lively#1538] [Ideas]
this might be outside the scope of this board but here go.
ive always loved the examples from the '**xscreensaver**' software available for android,

after playing around with **lively** im wondering if it would be possible to port some of the examples from xscreensaver to a webgl format to be used as a desktop background in lively.

mainly i just want the "**substrate**" program or something similar as a desktop background. i know a bit about "coding" etc and 

im wondering if it would be possibly to translate some of the source code to webgl or anything like that or if anyone with the knowhow would be interested in cooperating in an effort to do that.

hope this isnt too long winded, **thanks**

the software mentioned: https://www.jwz.org/xscreensaver/screenshots/

**Comment by katahiromz** [comment]
WebAssembly or handly porting.

You might have to have C and JavaScript knowledge.

**Reply by rocksdanister** [reply]
Your [Windows port](https://github.com/katahiromz/XScreenSaverWin) should work as [Application wallpaper.](https://github.com/rocksdanister/lively/wiki/Application-Wallpaper)

**Reply by kenwud** [reply]
woah! since i installed the windows store version i had no idea this feature even existed! defintely trying this out right now
 (
http://katahiromz.web.fc2.com/xscreensaverwin/eindex.html gives 404 error, but i found the binary on a mirror website)
         
for some reason the github lively version only lets me navigate the application with keyboard , it doesnt seem to respond to mouse input,
dragging and dropping a .scr or exe into the 'add wallpaper' window gives me a restricted mouse icon, let me know if im doing something wrong or if i should make this into a seperate issue thread, the windows store version now also not accepting mouse input
